metis-core/metis-core-common/src/main/java/eu/europeana/metis/core/workflow/plugins/OaipmhHarvestPlugin.java

```diff
@@ -130,14 +133,18 @@ public void setExternalTaskId(long externalTaskId) {
   }
 
   @Override
-  public ExecutionRecordsStatistics getExecutionRecordsStatistics() {
-    return executionRecordsStatistics;
+  public ExecutionProgress getExecutionProgress() {
+    return executionProgress;
   }
 
   @Override
-  public void setExecutionRecordsStatistics(
-      ExecutionRecordsStatistics executionRecordsStatistics) {
-    this.executionRecordsStatistics = executionRecordsStatistics;
+  public void setExecutionProgress(
+      ExecutionProgress executionProgress) {
+    this.executionProgress = executionProgress;
+  }
+
+  public String getTopologyName() {
```

Comment:
I don't think this method is used?

Reply:
It is required by the serialization to json. I will add a comment.